hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestSchedulerConfGenerator.java

```diff
@@ -34,7 +37,7 @@ public class TestSchedulerConfGenerator {
 
   @Test
   public void testGenerateSparkSchedulingConf() throws Exception {
-    HoodieDeltaStreamer.Config cfg = new HoodieDeltaStreamer.Config();
+    HoodieStreamer.Config cfg = new HoodieStreamer.Config();
```

Comment:
Why this change?

Reply:
`HoodieDeltaStreamer` is deprecated, so replace it in this test:
https://github.com/apache/hudi/blob/a0d83798251611428f70e5449f9ec338da3e7f15/hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java#L32-L37